napari/__main__.py

```diff
@@ -288,6 +292,12 @@ def _run():
         # it will collect it and hang napari at start time.
         # in a way that is machine, os, time (and likely weather dependant).
         viewer = Viewer()
+
+        # For backwards compatibility
+        # If the --stack option is provided without additional arguments
+        # just set stack to True similar to the previous store_true action
+        if args.stack and len(args.stack[0]) == 0:
```

Comment:
maybe here we should check if len of `args.stack` is one?

Reply:
Agreed, I added an additional check if `len(args.stack) == 1`